providers/fips/fipsprov.c

```diff
@@ -208,17 +210,19 @@ int OSSL_provider_init(const OSSL_PROVIDER *provider,
     if (ctx == NULL)
         return 0;
 
+    *out = fips_dispatch_table;
+    *provctx = ctx;
+
     /*
      * TODO(3.0): Remove me. This is just a dummy call to demonstrate making
      * EVP calls from within the FIPS module.
      */
-    if (!dummy_evp_call(ctx)) {
-        OPENSSL_CTX_free(ctx);
+    if (!dummy_evp_call(*provctx)) {
```

Comment:
Not really sure why this is necessary. We just created the ctx!

Reply:
It's still demonstrating making such EVP calls from within the FIPS module...

Reply:
Well...yeah....but....hmmmm....ok

Reply:
We can remove it at any time.  At your leasure, really, since you put it in to begin with...  However, that's not what this PR is about, so won't touch it here (apart from that obvious change).